src/styles/_App.scss

```diff
@@ -244,3 +244,18 @@
 hr {
   margin: 75px 0;
 }
+
+.section-header div:nth-child(2) {
+  align-self: flex-end;
+}
+
+@media (min-width: 768px) {
+  .section-button .text-right * {
+    margin-left: 3px;
+  }
+}
+@media (max-width: 767px) {
```

Comment:
can we add the break point to the variables file so we can reference everywhere?

Reply:
sure. I'll do that